### DISCUSSION TITLE
Migrar para distros Debian

### DISCUSSION THREAD

**Discussion by Ahosall** [Zian25/UniTools-Termux#5] [Q&A] [answered]
# Migrar para distros Debian

Olá boa tarde, gostaria de saber se eu posso refazer seu programa, alguns comandos dele (o pkg por exemplo) não funciona em algumas distros Linux, então eu pensei em migrar os comandos dele para o Debian (já que a maioria das distros são baseados nele).

Claro eu irei deixar os créditos também, só quero disponibilizar a ferramenta para que todas os usuários de Linux possam usar sem erros.

Gostaria de saber também se podemos manter contato, via WhatsApp, Discord, ou até mesmo E-Mail.

No Aguardo.

### Formas de contato ...
<b>Discord</b>: Feh's#5060
<b>E-Mail</b>: zen.wixyos.br@gmail.com

**Comment by Zian25** [comment, marked as answer]
Olá! Desculpe pela demora a responder, não recebi notificação. Sim, você pode refazer ao seu jeito ^w^, fica mais fácil agora com as atualizações de clean code kkkkkk, tentei adicionar pelo discord mas não consegui, mandei e-mail. :)

**Reply by Ahosall** [reply]
Ops, desculpe kkkkk esqueci de ativar essa função...

![image](https://user-images.githubusercontent.com/49027157/126319100-7591f987-dfe2-4670-a7b6-02e5e4385b79.png)

Certo eu irei atualizar o repo e começar a migrar o código. :>
